src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
+
+    public synchronized static Perfection getPerf() {
```

Comment:
Since is a implementation of the Singleton pattern, the method to get the instance should be renamed to getInstance() following the conventions and code readability.